imports/plugins/core/ui/client/components/table/sortableTable.js

```diff
@@ -272,6 +271,13 @@ class SortableTable extends Component {
   render() {
     const { ...otherProps } = this.props;
     const defaultClassName = "-striped -highlight";
+    const displayNoResultsFound = () => {
+      let displayText = "";
+      if (this.getTableData() === 0) {
+        displayText = <span className="sortableTable-noDataText">{this.props.noDataMessage}</span>;
+        return displayText;
```

Comment:
Should this return be defined outside that `if` so that a result is return regardless? I assume that's why you defined it outside the `if`?

Reply:
Also needs JSDocs?

Reply:
Oh yes the idea was to return display text even when the `if` condition isn't satisfied, I have rectified that now. Also, I have fixed the requested changes.